src/utils.rs

```diff
@@ -172,16 +155,12 @@ pub async fn get_webxdc_version(file: impl AsRef<Path>) -> anyhow::Result<i32> {
 #[derive(Clone, Copy, Type)]
 pub enum Webxdc {
     Shop,
-    Submit,
-    Review,
 }
```

Comment:
I only removed the variants here instead of refactoring all of them for now. This leaves us with an enum only having one variant, but refactoring are harder to undo later and it's not unlikely that we will support different webxdcs later.